nginx.conf.d/nginx.conf

```diff
@@ -66,6 +66,10 @@ http {
 
 		server_name webapp.##DOMAIN_NAME## webapp;
 
+		ssl_client_certificate /etc/volume/certs/cacert.pem;
+		ssl_verify_client		optional;
+		ssl_verify_depth		2;
```

Comment:
This will cause webapp to always ask for client certs. Don't we want a dedicated site only for client cert validation?